sql/sql_parse.cc

```diff
@@ -4573,6 +4573,10 @@ int mysql_execute_command(THD *thd, bool first_level) {
         break;
 
 #ifdef WITH_WSREP
+      if (thd->locked_tables_mode) {
```

Comment:
What is it for?

Reply:
This is needed if we want to block DB operation while are in locked_tables_mode.
```
LOCK TABLE test.p WRITE;
CREATE SCHEMA S; -> This query will fail on node1 with ER_LOCK_OR_ACTIVE_TRANSACTION, but will be successful on node2
```